tests/instancetype_test.go

```diff
@@ -518,7 +518,7 @@ var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-c
 			Expect(vmi.Spec.Domain.CPU.Sockets).To(Equal(originalInstancetypeCPUGuest))
 
 			By("Creating a second VirtualMachine using the now updated VirtualMachineInstancetype and original VirtualMachinePreference")
-			newVMI := tests.NewRandomVMI()
+			newVMI := libvmi.New()
 			removeResourcesAndPreferencesFromVMI(newVMI)
```

Comment:
FWIW https://github.com/kubevirt/kubevirt/pull/7549#discussion_r926477037 we can remove `removeResourcesAndPreferencesFromVMI` now `libvmi.New()` is used. I can follow up and clean this up later if you don't want to amend this into your change.

Reply:
Thanks for the note!
Done.